ibctest/relayer.go

```diff
@@ -19,13 +21,33 @@ import (
 type Relayer struct {
 	t *testing.T
 
-	home string
+	config RelayerConfig
+	home   string
 
 	// Set during StartRelayer.
 	errCh  chan error
 	cancel context.CancelFunc
 }
 
+// Build returns a relayer interface
+func NewRelayer(
+	t *testing.T,
+	config RelayerConfig,
+) ibc.Relayer {
+	r := &Relayer{
+		t:      t,
+		home:   t.TempDir(),
+		config: config,
+	}
+
+	res := r.sys().Run(zaptest.NewLogger(t), "config", "init", "--memo", config.Memo)
+	if res.Err != nil {
+		panic(fmt.Errorf("failed to rly config init: %w", res.Err))
```

Comment:
Is `t.Fatal` more appropriate than a panic here?

Reply:
done